src/js/tabs/vtx.js

```diff
@@ -101,19 +101,46 @@ TABS.vtx.initialize = function (callback) {
     // Emulates the MSP read from a vtxConfig object (JSON)
     function read_vtx_config_json(vtxConfig, vtxcallback_after_read) {
 
+        function checkArrayType(value, type) {
+
+            // Check each of its elements
+            if (Array.isArray(value)) {
+                value.forEach(function(element) {
```

Comment:
value.forEach((el) => checkType(el, type));
is more readable

Reply:
True, now I'm out of the computer, I will change it later

Reply:
It's not necessary. Just my 2 cent)

Reply:
Fixed!